apps/challenges/aws_utils.py

```diff
@@ -95,6 +95,96 @@
 }
 
 
+def scale_resources(challenge, new_cores, new_memory):
+    client = get_boto3_client("ecs", aws_keys)
+    if challenge.worker_cpu_cores == new_cores and challenge.worker_memory == new_memory:
+        message = "Error. Worker cores and memory were not modified."
+        return {
+            "Error": message,
+            "ResponseMetadata": {"HTTPStatusCode": HTTPStatus.BAD_REQUEST},
+        }
+    if not challenge.task_def_arn:
+        message = "Error. Task definition not yet registered for challenge{}.".format(
+            challenge.pk
+        )
+        return {
+            "Error": message,
+            "ResponseMetadata": {"HTTPStatusCode": HTTPStatus.BAD_REQUEST},
+        }
+    # deregister
```

Comment:
Can you make the comment more descriptive? Please take a look at other APIs for reference